src/command_modules/azure-cli-network/azure/cli/command_modules/network/_format.py

```diff
@@ -185,4 +185,23 @@ def transform_waf_rule_sets_table_output(result):
                 item_obj['ruleGroup'] = rule_group_name
                 transformed.append(item_obj)
     return transformed
-    
\ No newline at end of file
+
+
+def transform_network_usage_list(result):
+    result = list(result)
+    for item in result:
+        item.current_value = str(item.current_value)
+        item.limit = str(item.limit)
+        item.local_name = item.name.localized_value
```

Comment:
Yes. I wanted to preserve the SDK structure so that's why I didn't remove the value.